CMakeLists.txt

```diff
@@ -622,24 +644,6 @@ else()
     add_dependencies(doc doxygen-doc)
 endif()
 
-find_program(ASCIIDOC_PATH asciidoc)
-find_program(SOURCE_HIGHLIGHT_PATH source-highlight)
-if ("${ASCIIDOC_PATH}" STREQUAL "ASCIIDOC_PATH-NOTFOUND")
-    message(WARNING "asciidoc is not installed. Readme can not be built.")
-elseif("${SOURCE_HIGHLIGHT_PATH}" STREQUAL "SOURCE_HIGHLIGHT_PATH-NOTFOUND")
-    message(WARNING "source-highlight is not installed. Readme can not be built.")
-else()
-    message("asciidoc found")
-    message("source-highlight found")
-    add_custom_command(TARGET doc
-                       POST_BUILD
-                       COMMAND asciidoc
-                               -a version=${VSOMEIP_VERSION}
-                               -b html
-                               -o documentation/vsomeipUserGuide.html
-                               ${PROJECT_SOURCE_DIR}/documentation/vsomeipUserGuide)
-endif()
-
```

Comment:
Why was this removed?  I have some small changes that make it actually work (produce a PDF)

(formatting is different because I ran cmake-format on it)
```
find_program(ASCIIDOC_PATH asciidoc)
find_program(SOURCE_HIGHLIGHT_PATH source-highlight)
if("${ASCIIDOC_PATH}" STREQUAL "ASCIIDOC_PATH-NOTFOUND")
  message(WARNING "asciidoc is not installed. Readme can not be built. This may require a pip3 install asciidoc3")
elseif("${SOURCE_HIGHLIGHT_PATH}" STREQUAL "SOURCE_HIGHLIGHT_PATH-NOTFOUND")
  message(WARNING "source-highlight is not installed. Readme can not be built.")
else()
  message("asciidoc found")
  message("source-highlight found")
  add_custom_command(
    TARGET doc
    POST_BUILD
    COMMAND asciidoc -a version=${VSOMEIP_VERSION} -b html -o documentation/vsomeipUserGuide.html
            ${PROJECT_SOURCE_DIR}/documentation/vsomeipUserGuide
  )
endif()

find_program(A2X_PATH a2x)
if(NOT A2X_PATH)
  message(WARNING "a2x is not installed. Readme can not be built as a pdf.")
else()
  add_custom_command(
    TARGET doc
    POST_BUILD
    COMMAND
      a2x -v --no-xmllint -a version=${VSOMEIP_VERSION} -f pdf -a tabsize=4 --dblatex-opts
      '-P latex.output.revhistory=0 -P imagedata.default.scale=maxwidth=10cm,maxheight=8cm' -D
      ${PROJECT_BINARY_DIR}/documentation ${PROJECT_SOURCE_DIR}/documentation/vsomeipUserGuide
    BYPRODUCTS ${PROJECT_BINARY_DIR}/documentation/vsomeipUserGuide.pdf
  )
endif()
```

Reply:
We removed this because the documentation was moved to markdown

Reply:
oh, awesome.